content/en/blog/sysflow-falco-sidekick.md

```diff
@@ -0,0 +1,146 @@
+6. The attack completes when data is exfiltrated off site
+ 
+In this setup, the SysFlow monitors the Kubernetes cluster and outputs alerts to Falco Sidekick, which are annotated with an ID ("`sf.graphid`") referencing compact behavioral graphs labeled with MITRE ATT&CK metadata that can help diagnose alerts.
+
+As a proof-of-concept, we customized Falco Sidekick's web interface to fetch and display the graph associated with an event. For example, the screenshot below shows the precise infection point in which the benign node.js application has been hijacked into spawning an attack script. 
+
+<br/>
+
+![image description](/img/sysflow-falco-sidekick/04-falco-sidekick-ui.png)
+
+<br/>
+
+Based on this alert, we can further investigate the potential threat using our interactive Jupyter environment. Typically, the first step in this diagnosis routine is to craft a hypothesis, or indicator, based on the original event. The syntax follows the rules condition language used in the SysFlow policy engine. A reasonable hypothesis would be to query for network and filesystem activity associated with processes containing the string "exfil" in its command line.
+ 
+Applying this indicator fetches a new graph from the storage backend that shows that the attack script was executed and has been busy interacting with the filesystem and communicating over the network.
+ 
+ Suspicious, huh?
+
+![image description](/img/sysflow-falco-sidekick/05-diagram-exfil-command.png)
+
+Since data exfiltration is of immediate concern, we can use the SysFlow APIs to quickly glance over the ingress and egress traffic associated with this process.
+We immediately recognize a periodic pattern in the network flows, which is indicative of command-and-control behavior, such as beaconing. We also see a suspiciously high outbound traffic at the end of the communication, which surely deserves further attention. 
+
+![image description](/img/sysflow-falco-sidekick/06-plot-c2-flow.png)
+
+
+We close our investigation by projecting the SysFlow trace associated with the above graph across different components, including container and network primitives. Two types of flows are of particular interest to us here: the backend flow from 172.30.106.11:3000 (internal object storage endpoint) that connects to the egress flow 104.244.42.*:443 (external exfiltration endpoint) through container bc3e. 
+
+![image description](/img/sysflow-falco-sidekick/07-graph-exfiltration-flow.png)
+
+### Conclusion 
+
+This integration is still experimental, but we are excited about the possibilities it holds and reminds us how well the Falco architecture makes it possible to use it in the creative way that is SysFlow. The goal is to refine this proof-of-concept to inform our future contributions to Falco. For example, a natural next step is to work on a SysFlow library and plugin for the incubating [libs plugin system](https://github.com/falcosecurity/plugins). 
```

Comment:
This integration is still experimental, but we are excited about the possibilities it holds and reminds us how well the Falco architecture makes it possible to use it in a creative way that is SysFlow. The goal is to refine this proof-of-concept to inform our future contributions to Falco. For example, a natural next step is to work on a SysFlow library and plugin for the incubating [libs plugin system](https://github.com/falcosecurity/plugins).